README.md

```diff
@@ -3,741 +3,470 @@
+ - Claimants are confident that the allocator is sound and will not leave the resource lock underallocated.
+ - Sponsors are confident that the allocator will not unduly censor fully allocated requests.
+ - Sponsors are confident that the arbiter is sound and will not process claims where the conditions were not successfully met.
+ - Claimants are confident that the arbiter is sound and will not *fail* to process claims where the conditions *were* successfully met.
+
+## Key Concepts
+
+### Resource Locks
+Resource locks are the fundamental building blocks of The Compact protocol. They are created when a depositor places tokens (either native tokens or ERC20 tokens) into The Compact. Each resource lock has four key properties:
+
+1.  The **underlying token** held in the resource lock.
+2.  The **allocator** tasked with cosigning on claims against the resource locks (see [Allocators](#allocators)).
+3.  The **scope** of the resource lock (either spendable on any chain or limited to a single chain).
+4.  The **reset period** for forcibly exiting the lock (see [Forced Withdrawals](#forced-withdrawals)) and for emissary reassignment timelocks (see [Emissaries](#emissaries)).
+
+Each unique combination of these four properties is represented by a fungible ERC6909 tokenID. The owner of these ERC6909 tokens can act as a sponsor and create compacts.
+
+The `scope`, `resetPeriod`, and the `allocatorId` (obtained when an allocator is registered) are packed into a `bytes12 lockTag`. A resource lock's specific ID (the ERC6909 `tokenId`) is a hash of this `lockTag` and the `token` address. This `lockTag` is used throughout various interfaces to succinctly identify the parameters of a lock.
+
+**Fee-on-Transfer and Rebasing Token Handling:**
+-   **Fee-on-Transfer:** The Compact correctly handles fee-on-transfer tokens for both deposits and withdrawals. The amount of ERC6909 tokens minted or burned is based on the *actual balance change* in The Compact contract, not just the specified amount. This ensures ERC6909 tokens accurately represent the underlying assets.
+-   **Rebasing Tokens:** **Rebasing tokens (e.g., stETH) are NOT supported in The Compact V1.** Any yield or other balance changes occurring *after* deposit will not accrue to the depositor's ERC6909 tokens. For such assets, use their wrapped, non-rebasing counterparts (e.g., wstETH) to avoid loss of value.
+
+### Allocators
+Each resource lock is mediated by an **allocator**. Their primary responsibilities include:
+1.  **Preventing Double-Spending:** Ensuring sponsors don't commit the same tokens to multiple compacts or transfer away committed funds.
+2.  **Validating Transfers:** Attesting to standard ERC6909 transfers of resource lock tokens (via `IAllocator.attest`).
+3.  **Authorizing Claims:** Cosigning on claims against resource locks (via `IAllocator.authorizeClaim` or by providing off-chain `allocatorData`).
```

Comment:
Probably want to avoid calling out co-signing explicitly; better to just refer to it as authorizing claims